packages/php-wasm/compile/Makefile

```diff
@@ -1,4 +1,30 @@
+# A phony target is one that is not really the name of a file; rather it is
+# just a name for a recipe to be executed when you make an explicit request.
+# There are two reasons to use a phony target: to avoid a conflict with a
+# file of the same name, and to improve performance.
+
+# If you write a rule whose recipe will not create the target file,
+# the recipe will be executed every time the target comes up for remaking.
+# https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
+.PHONY: all libz libzip libpng16 libxml2 libopenssl libsqlite3 libiconv libncurses libedit bison2.7 oniguruma base-image clean
+
+all: libz libzip libpng16 libxml2 libopenssl libsqlite3 libiconv libncurses libedit bison2.7 oniguruma base-image
+
+clean:
+	rm -rf ./libz/dist
+	rm -rf ./libzip/dist
+	rm -rf ./libpng16/dist
+	rm -rf ./libxml2/dist
+	rm -rf ./libopenssl/dist
+	rm -rf ./libsqlite3/dist
+	rm -rf ./libiconv/dist
+	rm -rf ./libncurses/dist
+	rm -rf ./libedit/dist
+	rm -rf ./bison2.7/dist
+	rm -rf ./oniguruma/dist
+
```

Comment:
these don't need to appear first. without any arguments, `make` will run the first target, which is why `all` is a common first target. `clean` doesn't need to move unless there's a reason to do it.